lib/ex_unit/test/ex_unit/doc_test_test.exs

```diff
@@ -507,6 +517,22 @@ defmodule ExUnit.DocTestTest do
     assert capture_io(fn -> ExUnit.run end) =~ "1 test, 0 failures"
   end
 
+  test "IEx prompt contains host" do
+    msg =
```

Comment:
It doesn't hurt to use `message` here (we already do this in the place where we raise the message). :bowtie: